src/Snakemake/rules/QC/fastqc.smk

```diff
@@ -30,10 +30,13 @@ rule fastqcR1:
         outdir="qc/{sample}/",
     log:
         "logs/qc/fastqc/{sample}_R1.log",
-    singularity:
-        config["singularity"]["fastqc"]
+    threads: 10
+    container:
+        config["singularity"].get("fastqc", config["singularity"].get("default", ""))
+    # wrapper:
+    #    "0.38.0/bio/fastqc"
     shell:
-        "(fastqc --quiet --outdir {params.outdir} {input}) &> {log}"
+        "(fastqc --quiet -t {threads} --outdir {params.outdir} {input}) &> {log}"
```

Comment:
What's wrong with the wrapper?

Reply:
Have had lots of problems with fastqc running forever. I don't think the one in the docker works. So when testing this I might have removed the wrapper.

Reply:
ok

Reply:
We could set it as an enhancement to change to a wrapper, which could be a good task for Linköping.